### DISCUSSION TITLE
Error in installation of foundary

### DISCUSSION THREAD

**Discussion by karthiksathishjeemain** [Cyfrin/foundry-full-course-cu#892] [Q&A]
While installing foundary, I am stuck in this error:

`karthik_sathish@KARTHIKSATHISH:~/hh-fcc/ethers-simple-storage$` curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could `not`
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

**Comment by theirrationalone** [comment]
Please install openssh and then try again!

**Comment by karthiksathishjeemain** [comment]
[
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/115690993/dfbbd109-18d8-4b71-8f3d-3bd44dd6dd0f)
](url)
bro still not possible

**Comment by Shiva-Sai-ssb** [comment]
Hey @karthiksathishjeemain, Attempt to resolve the problem by trying out the solutions offered in this link.

https://stackoverflow.com/questions/24611640/curl-60-ssl-certificate-problem-unable-to-get-local-issuer-certificate
